model-optimizer/mo/utils/ir_reader/layer_to_class.py

```diff
@@ -233,6 +233,21 @@ def assign_add_output_result(op: Node):
     op.out_port(0).connect(tmp_result.in_port(0))
 
 
+def topk_add_output_result(op: Node):
```

Comment:
Please, change the name of the variable from `op` to `node`

Reply:
Do we really need this change? We use `op` name inside all functions in this file.

Reply:
Remove this function completely. Just use it `TopKNormalizer.normalize_outputs` in line 275.